src/main/java/com/synopsys/integration/alert/channel/azure/boards/service/AzureCustomFieldManager.java

```diff
@@ -69,23 +102,31 @@ public AzureCustomFieldManager(String organizationName, AzureProjectService proj
     }
 
     public void installCustomFields(String projectName, String workItemTypeName) throws AlertException {
-        Future<ProjectWorkItemFieldModel> topLevelKeyFieldCreationResultHolder =
-            executorService.submit(() -> findOrCreateAlertCustomProjectField(projectName, ALERT_TOP_LEVEL_KEY_FIELD_NAME, ALERT_TOP_LEVEL_KEY_FIELD_REFERENCE_NAME, ALERT_TOP_LEVEL_KEY_FIELD_DESCRIPTION));
-        Future<ProjectWorkItemFieldModel> componentLevelKeyFieldCreationResultHolder =
-            executorService.submit(() -> findOrCreateAlertCustomProjectField(projectName, ALERT_COMPONENT_LEVEL_KEY_FIELD_NAME, ALERT_COMPONENT_LEVEL_KEY_FIELD_REFERENCE_NAME, ALERT_COMPONENT_LEVEL_KEY_FIELD_DESCRIPTION));
+        installCustomFields(projectName, workItemTypeName, AZURE_CUSTOM_FIELDS);
```

Comment:
Should we just remove the private method and put the contents in the public one?